config/operator/rbac/role.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: cl-operator-manager-role
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
+  - namespaces
+  - pods
+  - services
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ""
+  resources:
```

Comment:
why do we need a Persistent volume claim?

Reply:
This is how the controlplane is implemented for mapping the K8s secrets.